app/components/elements/forms/repeatable_nested_component.rb

```diff
@@ -6,14 +6,15 @@ module Forms
     # NOTE: Use the `NestedComponentPresenter` to invoke this component; do not directly instantiate it.
     class RepeatableNestedComponent < ApplicationComponent
       def initialize(form:, model_class:, field_name:, form_component:, hidden_label: false, bordered: true, # rubocop:disable Metrics/ParameterLists
-                     reorderable: false, label: nil)
+                     reorderable: false, single_field: false, label: nil)
```

Comment:
I tried to avoid adding another param, but because we can't rely on the number of fields in the form--in the case of keywords there are additional hidden fields--ended having to take this route.

Reply:
@lwrubel the struggle is real! Thanks for the rationale. 😄